genie/database_to_staging.py

```diff
@@ -1924,8 +1924,9 @@ def create_link_version(
             release_file["name"] != "data_clinical.txt" or release_type == "consortium"
         )
         is_gene_panel = release_file["name"].startswith("data_gene_panel")
+        is_depreciated_file = release_file["name"] in ["data_fusions.txt"]
```

Comment:
This is great approach!  If we need a systematic way of handling deprecated files in the future, we can build off of this.